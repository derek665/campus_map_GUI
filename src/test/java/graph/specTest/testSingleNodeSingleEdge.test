# test when a single node has a circular edge (same node for parent and child)

# Create a graph
CreateGraph circular

# Add single node
AddNode circular n1

# Add an edge to n1
AddEdge circular n1 n1 e1

# Print the nodes in graph
ListNodes circular

# Print the children in graph
ListChildren circular n1