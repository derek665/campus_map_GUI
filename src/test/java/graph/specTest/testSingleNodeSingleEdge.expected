# test when a single node has a circular edge (same node for parent and child)

# Create a graph
Created graph circular

# Add single node
added node n1 to circular

# Add an edge to n1
added edge e1 from n1 to n1 in circular

# Print the nodes in graph
circular contains: n1

# Print the children in graph
the children of n1 in circular are: n1(e1)